image_client/sql_csv_utils.py

```diff
@@ -93,7 +97,7 @@ def create_insert_statement(col_list: list, val_list: list, tab_name: str):
 
     return sql
 
-def insert_table_record(connection, logger_int , sql):
+def insert_table_record(connection, logger_int , sql, sqlite = False):
```

Comment:
since this was a util, not a class method, added a hidden sqlite arg , as sqlite connections don't work with get_cursor() only with cursor. I could change this if need be, try to monkeypatch it instead or create a derived testing function.